cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   template:
     metadata:
       labels:
-        app: cost-analyzer
```

Comment:
Can we leave this on as one of the selectorlabels to support things outside of the helm chart that may have used this label? Let's move this to _helpers.tpl

Reply:
You mean in common labels? I see no reason to keep it in selector labels as all resources referenced by the selectors are in the helm chart.

Reply:
I believe common labels is fine, we just don't want to remove this entirely from labels as people may be relying on it still.

Reply:
I add the `app` label again.